wrangler-storage/src/test/java/io/cdap/wrangler/store/recipe/RecipeStoreTest.java

```diff
@@ -176,4 +176,22 @@ public void testListRecipesPagination() {
     Assert.assertEquals(values.get(0), recipeRow3.getRecipe());
     Assert.assertEquals(page2.getNextPageToken(), "");
   }
+
+  @Test(expected = RecipeNotFoundException.class)
```

Comment:
Can you add one more test case of adding a recipe with the same name and namespace as deleted recipe?